jina/helper.py

```diff
@@ -1006,13 +1007,13 @@ def get_or_reuse_loop():
 
     :return: A new eventloop or reuse the current opened eventloop.
     """
+    _update_policy()
     try:
-        loop = asyncio.get_running_loop()
+        loop = asyncio.get_event_loop()
```

Comment:
this is the relevant change. We should use any loop (unless closed as checked in the next line).